src/uu/truncate/src/truncate.rs

```diff
@@ -27,78 +28,100 @@ enum TruncateMode {
+        .version(VERSION)
+        .about(ABOUT)
+        .usage(&usage[..])
+        .after_help(&long_usage[..])
+        .arg(
+            Arg::with_name(OPT_NO_CREATE)
+            .short("c")
+            .long(OPT_NO_CREATE)
+            .help("do not create files that do not exist")
+        )
+        .arg(
+            Arg::with_name(OPT_IO_BLOCKS)
+            .short("o")
+            .long(OPT_IO_BLOCKS)
+            .help("treat SIZE as the number of I/O blocks of the file rather than bytes (NOT IMPLEMENTED)")
+        )
+        .arg(
+            Arg::with_name(OPT_REFERENCE)
+            .short("r")
+            .long(OPT_REFERENCE)
+            .help("base the size of each file on the size of RFILE")
+            .value_name("RFILE")
+        )
+        .arg(
+            Arg::with_name(OPT_SIZE)
+            .short("s")
+            .long("size")
+            .help("set or adjust the size of each file according to SIZE, which is in bytes unless --io-blocks is specified")
+            .value_name("SIZE")
+        )
+        .arg(Arg::with_name(OPT_FILES).multiple(true).takes_value(true).min_values(1))
+        .get_matches_from(args);
+
+    let files: Vec<String> = matches
+        .values_of(OPT_FILES)
+        .map(|v| v.map(ToString::to_string).collect())
+        .unwrap_or_default();
 
-    if matches.opt_present("help") {
-        println!("{} {}", NAME, VERSION);
-        println!();
-        println!("Usage:");
-        println!("  {} [OPTION]... FILE...", NAME);
-        println!();
-        print!(
-            "{}",
-            opts.usage("Shrink or extend the size of each file to the specified size.")
-        );
-        println!(
-            "
-SIZE is an integer with an optional prefix and optional unit.
-The available units (K, M, G, T, P, E, Z, and Y) use the following format:
-    'KB' =>           1000 (kilobytes)
-    'K'  =>           1024 (kibibytes)
-    'MB' =>      1000*1000 (megabytes)
-    'M'  =>      1024*1024 (mebibytes)
-    'GB' => 1000*1000*1000 (gigabytes)
-    'G'  => 1024*1024*1024 (gibibytes)
-SIZE may also be prefixed by one of the following to adjust the size of each
-file based on its current size:
-    '+'  => extend by
-    '-'  => reduce by
-    '<'  => at most
-    '>'  => at least
-    '/'  => round down to multiple of
-    '%'  => round up to multiple of"
-        );
-    } else if matches.opt_present("version") {
-        println!("{} {}", NAME, VERSION);
-    } else if matches.free.is_empty() {
-        show_error!("missing an argument");
+    if files.is_empty() {
+        show_error!("Missing an argument");
         return 1;
     } else {
-        let no_create = matches.opt_present("no-create");
-        let io_blocks = matches.opt_present("io-blocks");
-        let reference = matches.opt_str("reference");
-        let size = matches.opt_str("size");
+        let no_create = matches.is_present(OPT_NO_CREATE);
+        let io_blocks = matches.is_present(OPT_IO_BLOCKS);
```

Comment:
Again, alphabetic order unless there is another underlying logical order or dependency.